util/qrcodes/qrcodes.go

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) Tailscale Inc & AUTHORS
+// SPDX-License-Identifier: BSD-3-Clause
+
+// Package qrcodes formats QR codes according to the active terminal.
+package qrcodes
+
+import (
+	"fmt"
+	"io"
+	"strings"
+
+	qrcode "github.com/skip2/go-qrcode"
+)
+
+// --qr-format values
+type Format string
+
+const (
+	FormatAuto  Format = "auto"
+	FormatASCII Format = "ascii"
+	FormatLarge Format = "large"
+	FormatSmall Format = "small"
+)
+
+// Fprintln formats s according to [Format] and writes to w and a newline is
+// appended. It returns the number of bytes written and any write error
+// encountered.
```

Comment:
might mention "QR Code" in here somewhere.  "and writes a QR Code to w, along with a newline"